romancal/pipeline/exposure_pipeline.py

```diff
@@ -187,10 +185,8 @@ def process(self, input):
         #          observations. This should not occur on-prem
         if result.meta.exposure.type == "WFI_IMAGE":
             if file_type == "asdf":
+                # FIXME run tweakreg?
                 result.meta.cal_step.tweakreg = "SKIPPED"
```

Comment:
FWIW here, I'm a bit confused by setting the step as "skipped" after self.tweakreg(tweakreg_input) below, but I suspect what's going on here is that we usually want to run tweakreg with the full 18 detectors rather than an individual file.  I actually haven't seen the SDP code that controls that.

@ddavis-stsci , can you clarify the logic here?  Thanks!

Reply:
Resolving due to lack of response.